pkg/config/constants/constants.go

```diff
@@ -21,6 +21,15 @@ const (
 	// AuthCertsPath is the path location for mTLS certificates
 	AuthCertsPath = "/etc/certs/"
 
+	// PilotWellKnownDNSCertPath is the path location for Pilot dns serving cert, often used with custom CA integrations
+	PilotWellKnownDNSCertPath   = "/var/run/secrets/istiod/tls/"
```

Comment:
```suggestion
	PilotWellKnownDNSCertPath   = "./var/run/secrets/istiod/tls/"
```

To avoid needing root

Reply:
ah nice, so the workdir of the pod is always the filesystem root? is that why this would work in both contexts?

Reply:
Yeah we make this assumption in a variety of places; for non-k8s it allows running in other directories if desired. Nice for testing as well